stopify-continuations/src/exposeGettersSetters.ts

```diff
@@ -1,3 +1,15 @@
+/**
+ * This transformation allows for getters and setters support in stopify.
+ * Getters and setters are implicity called on object accesses. This
+ * transform makes it such accesses explicit:
+ *
+ * Getters: a.b        => $gs.get_prop(a, "b")
+ * Setters: a.b = v    => $gs.get_prop(a, "b", v)
+ * Delete:  delete a.b => $gs.delete_prop(a, "b")
+ *
+ * The functions simply access the property. The benefit of making this a
```

Comment:
I did not get it at first. This is very clever.

Reply:
I bet I can make it cleaner by doing `$gs.get_prop(() => a.b)`. This will also side step all the stupid prototype issues.